### PR TITLE
updated CONTRIBUTE.md

### PR DESCRIPTION
**Pull request by grd**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,5 +69,8 @@ We have also automated the process via [clang-format](http://clang.llvm.org/docs
 
 [How to clang-format a diff](http://clang.llvm.org/docs/ClangFormat.html#script-for-patch-reformatting)
 
+## Keeping up to date with the master branch
```
Comment (sevki) [inline CONTRIBUTING.md]: can you carry this section up to submitting a patch and make it a subheader with `###`. it breaks the narrative when you have coding conventions in the middle

**Comment by sevki** [comment]
you might want to consider adding `closes #66 ` to your commit message if you think it resolves that issue

**Comment by grd** [comment]
I forked the repo again to test it. In Ubuntu you can't `git clone git@github.com:Harvey-OS/harvey.git` instead you have to do `git clone https://github.com/Harvey-OS/harvey.git`. Is this also in other platforms? I think it would be better to change that line. Also the part of adding a `git remote` could be useful.

**Comment by floren** [comment]
I haven't had any problems cloning using ssh rather than https; did you upload your machine's SSH pubkey to Github? I just checked out Harvey using ssh and it worked fine.

**Comment by grd** [comment]
Okay, that's probably it. I leave it as is.

**Comment by sevki** [comment]
Hey @grd, I'm sorry you closed this pull request because I liked the conversation we were having, it showed me that we are missing better documentation since we failed to get your PR merged.

Harvey's wiki, while aiming to be a good starting point is just that, it isn't the definitive guide on the subject, that's why it links to the [pull requests](http://help.github.com/pull-requests/) and [Github Flow](https://guides.github.com/introduction/flow/), which are the definitive guides, and are constantly updated.I currently think that entire section is drawing attention from those guides so I'd like it if they went away.

And what I didn't realize at the time of writing this was If you do a search on google or youtube about Git and/or GitHub you will see stuff like this webcast about [pull requests](https://www.youtube.com/watch?v=81uKcXZoQ2A) and this one about [workflows](https://www.youtube.com/watch?v=EwWZbyjDs9c) that cover the entire topic exhaustively, which is an indication that it is not as straight forward as I thought it was when I wrote that section.

So if you'd like to re-open this pull request and continue that conversation, and we can write a better doc you'll certainly have my +1

**Comment by grd** [comment]
O sorry, I thought it was closed because of the closes #66 comment. I can't reopen this because I removed my repo and forked it again (stupid github) so ... create a new issue or a new pull request?

**Comment by sevki** [comment]
@grd that'd be great.

**Comment by grd** [comment]
That will be tomorrow.

**Comment by elbing** [comment]
Awaiting nicely :-)

**Pull request closed without merging**